batch.go

```diff
@@ -228,16 +236,25 @@ func (p *ProcessingProfile) processPlotDefs(ctx context.Context, cfg *PlotConfig
 		fnames []string
 		err    error
 	)
+
+	matchGlob := "*.yaml"
+
 	if p.SourceIsDir() {
 		slog.Info("using plot definitions in " + p.Source)
 		infs = os.DirFS(p.Source)
-		fnames, err = fs.Glob(infs, "*.yaml")
-		if err != nil {
-			return fmt.Errorf("failed to read input directory: %w", err)
-		}
+		// fnames, err = fs.Glob(infs, "*.yaml")
 	} else {
 		infs = os.DirFS(filepath.Dir(p.Source))
-		fnames = []string{filepath.Base(p.Source)}
+		matchGlob = filepath.Base(p.Source)
+		// fnames = []string{filepath.Base(p.Source)}
+	}
+	if cfg.MatchGlob != "" {
+		fnames, err = fs.Glob(infs, cfg.MatchGlob)
+	} else {
+		fnames, err = fs.Glob(infs, matchGlob)
```

Comment:
Could put this `matchGlob` into the value of the `StringFlag` option above. Then you could skip this if statement here and always just use `cfg.MatchGlob`

Reply:
I don't think so, matchGlob is different when the source is a single file (it is set to the filename)

Reply:
You're right; apologies for the noise!